frontend/src/scenes/trends/personsModalLogic.ts

```diff
@@ -0,0 +1,16 @@
+import { kea } from 'kea'
+import { personsModalLogicType } from './personsModalLogicType'
+
+export const personsModalLogic = kea<personsModalLogicType>({
+    actions: () => ({
+        setSearchTerm: (term: string) => term,
+    }),
+    reducers: () => ({
+        searchTerm: [
+            '',
+            {
+                setSearchTerm: (_, term) => term,
+            },
+        ],
+    }),
+})
```

Comment:
Let's keep action payloads as objects. See the last gray box in the [kea docs actions](https://kea.js.org/docs/guide/concepts#actions) section for more details :).